pkg/integrations/splunk/splunk.go

```diff
@@ -59,6 +60,8 @@ func ProcessSplunkHecIngestRequest(ctx *fasthttp.RequestCtx, myid uint64) {
 		}
 	}
 
+	//update usage stats
```

Comment:
Is there something missing here? You have added a comment of `// Update usage stats`, but the following lines do not do updating of usage stats.